Bugzilla/DB.pm

```diff
@@ -111,8 +111,11 @@ sub connect_shadow {
     my $connect_params = dclone(Bugzilla->localconfig);
     $connect_params->{db_host} = Bugzilla->get_param_with_override('shadowdbhost');
     $connect_params->{db_name} = Bugzilla->get_param_with_override('shadowdb');
-    $connect_params->{db_port} = Bugzilla->get_param_with_override('shadowport');
-    $connect_params->{db_sock} = Bugzilla->get_param_with_override('shadowsock');
+    $connect_params->{db_port} = Bugzilla->get_param_with_override('shadowdbport');
```

Comment:
notice that `shadowport` and `shadowsock` below have been changed to `shadowdbport` and `shadowdbsock`. I assume this was a typo from before, these new values match what is defined in the localconfig.

Reply:
looks like sock got deleted here?